src/components/shared/CategoriesList.vue

```diff
@@ -1,31 +1,15 @@
 <template>
   <div>
-    <div class="bh-grid bh-grid--inline hide-mobile">
-      <div
-        v-for="category in categories"
-        :key="category.id"
-        class="bh-grid__item">
-        <bh-button
-          :primary="val === category.id"
-          @click="select(category.id)">{{ category.name }}</bh-button>
-      </div>
-    </div>
-    <div class="hide-desktop">
-      <div class="bh-grid">
-        <div class="bh-grid__item">
-          <bh-dropdown>
-            <bh-button
-              slot="trigger"
-              primary>
-              <span>{{ selectedCategoryName }}</span>
-              <bh-icon value="menu-down" />
-            </bh-button>
-            <bh-dropdown-item
-              v-for="category in categories"
-              :key="category.id"
-              @click="select(category.id)">{{ category.name }}</bh-dropdown-item>
-          </bh-dropdown>
-        </div>
+    <div class="bh-grid categories-list">
+      <div class="bh-grid__item text-right">
+        <bh-dropdown
+          :title="selectedCategoryName"
```

Comment:
Maybe rename this variable `selectedCategoryName` to `dropdownTitle` is more direct.